test/ng/directive/ngOptionsSpec.js

```diff
@@ -1130,7 +1130,7 @@ describe('ngOptions', function() {
       }).not.toThrow();
     });
 
-    it('should re-render if a propery of the model is changed when using trackBy', function() {
+    it('should re-render if the tracked property of the model is changed when using trackBy', function() {
```

Comment:
We are not actually interested in watching any old property on the model. The important thing is the `track by` expression.